Core/Src/freertos.c

```diff
@@ -189,6 +192,68 @@ __NO_RETURN void sendMotorMsg(void *argument) {
     }
 }
 
+
+typedef union IntBytes {
+	uint16_t int_value;
+	uint8_t bytes[2];
+} IntBytes;
+
+#define MAX_NUM_16_BIT 65536
+#define MAX_FLOAT_50B 100
+#define MAX_FLOAT_503 100
+#define MAX_FLOAT_502 300
+
+void addFloat(uint8_t* data, uint32_t max) {
```

Comment:
Add documentation for what each function does, details about what the arguments are, any preconditions/postconditions, etc.